projects/ntpsec/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd fuzz
+make all
+
+cp FuzzClient $OUT/FuzzClient
+cp FuzzExtens $OUT/FuzzExtens
+cp FuzzServer $OUT/FuzzServer
+cp FuzzLeapsec $OUT/FuzzLeapsec
+
+cp input/FuzzClient_seed_corpus.zip $OUT/FuzzClient_seed_corpus.zip
```

Comment:
I don't want zip files in this repo anymore. Please download the zips from some place else instead of comitting them (can even be from https://github.com/google/oss-fuzz/raw/c17c52fc0efab08355c565f37972fb0a1331b94c/projects/ntpsec/fuzz/input/FuzzClient_seed_corpus.zip)